rusoto/services/acm-pca/Cargo.toml

```diff
@@ -32,6 +32,7 @@ default-features = false
 
 [features]
 default = ["native-tls"]
+deserialize_structs = ["bytes/serde"]
```

Comment:
If I understand this correctly it enables `serde` support for the [`bytes crate`](https://github.com/tokio-rs/bytes). This doesn't appear [to be exported](https://github.com/tokio-rs/bytes/blob/master/src/lib.rs#L97) from `bytes` so it's for internal use of that crate.

Is this change required because Rusoto types use `bytes` internally? The other option being leaving the feature definition the empty set `[]` like `serialize_structs` was before.

Reply:
Indeed it is. Without that feature being specified, any struct with a `bytes` field will not build with `Serialize` or `Deserialize` derived. The first example of that I hit while working on this was [here](https://github.com/rusoto/rusoto/pull/1639/files?file-filters%5B%5D=.rs#diff-ec8eff15ba69bc3a2470637f6f67f6edR2070). 

I'm not that up on the inner workings of the Bytes crate and macros, but the feature there appears to activate [this module](https://github.com/tokio-rs/bytes/blob/master/src/serde.rs), which triggers the implementations of `Serialize` and `Deserialize` for various types within the crate.

I did look into trying to only add the feature to the dependent list on the crates that need it, but I noticed that `bytes` is included in all of the service crates, so presumably somebody already decided that it was too hard or not worth the trouble to figure out which crates actually need it.